CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+## How to contribute to software.opensuse.org
+
+#### **Did you find a bug?**
+
+* **Do not open up a GitHub issue if the bug is a security vulnerability
+  in the application**, and instead contact the [security team](mailto:security-team@suse.de).
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/openSUSE/software-o-o/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/openSUSE/software-o-o/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, including screenshots of what you see in your browser. You can attach screenshots to the text by drag and dropping them.
+
+#### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+#### **Did you fix whitespace, format code, or make a purely cosmetic patch?**
+
+With the rubocop version we are pinned to, there should be no warnings, and the only way to fix "pending" violations we have with rules is to disable some "ignored" issues in '.rubocop_todo.yml', re-run rubocop, fix those issues, and commit both '.rubocop_todo.yml' and the fixes at the same time.
```

Comment:
I would add that a commit per cop should be done, but I can send a PR to modify it, so that you don't need to make changes on this. :thinking:

Reply:
I think it is a good idea., both the idea about the "cop" and making it a separate PR.